Docker.DotNet/Models/HostConfig.cs

```diff
@@ -12,6 +12,9 @@ public class HostConfig
         [DataMember(Name = "Links")]
         public IList<string> Links { get; set; }
 
+        [DataMember(Name = "GroupAdd")]
+        public IList<string> GroupAdd { get; set; }
```

Comment:
Hmm alright then, I'm guessing you really need this field. :) Also please consider making a docs PR to Docker.